int32 dummy
---
int64 len
float32 resolution